packages/blade/src/components/Card/CardContext.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type CardContextType = true | null;
```

Comment:
We do need `null | true`

Say we want to check if `CardHeaderCounter` is inside `Card` we will call the `useInsideCard` hook in `CardHeaderCounter`. 

Now if the provider value is not initialised or null we will throw the error. But if we pass `true` and we get back the context's value that means we are inside card. 


```tsx
const useInsideCard = (componentName: string): CardContextType => {
  const context = React.useContext(CardContext);
  // ^ If this hook is called inside Card then it will return `true` 
  // But if we put the context's value as null only then this will return null and we will throw error even tho it's okay
  if (!context) {
    throw new Error(`[Blade Card]: ${componentName} cannot be used outside of Card component`);
  }
  return context;
};
```

Reply:
correct that's waht i'm saying if context exists then just return `true` because we don't really use `context` value anywhere right?

Reply:
Oh you mean here? 

```
  return context;
```

Yeah right we can do that.

Reply:
Done.